### DISCUSSION TITLE
Auto-rotate rotates skybox ok, if model moved, skybox reflection is stuck on some materials

### DISCUSSION THREAD

**Discussion by sprengerst** [google/model-viewer#1951] [Q&A]
Hello,

we have a problem with an auto rotating model:
http://vr.mazing-ar.com/demos/bugreport

If it rotates the wooden texture is corresponding in a correct way to the skybox and is getting darker and brighter.

Bright:
![image](https://user-images.githubusercontent.com/6854862/104008666-04197700-51aa-11eb-8fd0-ac0b1be4fe4d.png)

Darker:
![image](https://user-images.githubusercontent.com/6854862/104008695-0da2df00-51aa-11eb-8313-b5104abdb37c.png)

So far so good. But if we drag the object in the "dark  state" the texture is not responding anymore. So it stays dark. Regardless of the manual triggered rotation the wood stays dark.

![image](https://user-images.githubusercontent.com/6854862/104008794-3b882380-51aa-11eb-8b93-5bbe9e5d05f6.png)

**Comment by elalish** [comment]
This isn't exactly a bug, but does show some undesirable behavior we're fixing. When the object auto-rotates, it rotates relative to its environment (this is by design), while user interaction moves the camera (akin to moving the object and its environment together). The dark vs. bright is because our default environment is not very evenly lit, which is being fixed in #1761.

**Comment by sprengerst** [comment]
Thank you very much for this Information and your work. Looking forward to the next release.
